worlds/tracker/TrackerClient.py

```diff
@@ -640,6 +662,13 @@ def on_package(self, cmd: str, args: dict):
                     asyncio.create_task(wait_for_items(self),name="UT Delay function") #if we don't get new items, delay for a bit first
                 self.watcher_task = asyncio.create_task(game_watcher(self), name="GameWatcher") #This shouldn't be needed, but technically 
             elif cmd == 'RoomUpdate':
+                if self.items_handling != ITEMS_HANDLING:
+                    unknown_locations = [location for location in self.checked_locations
+                                         if location not in self.locations_info]
+                    if len(unknown_locations) > 0:
```

Comment:
```suggestion
                    unknown_locations:
```
(could also be moved to a function and called in both of these places